cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/util/HttpUrlManager.kt

```diff
@@ -119,19 +119,21 @@ internal object HttpUrlManagerSingleton : HttpUrlManager {
             .build()
     }
 
-    private fun urlPmCcpa(pmConf: PmUrlConfig, env: Env, messSubCat: MessageSubCategory): HttpUrl {
-
-        // ott: https://cdn.privacy-mgmt.com/ccpa_ott/index.html?message_id=527843
-        //      https://ccpa-notice.sp-stage.net/ccpa_pm/index.html?message_id=14777
+    private fun urlPmCcpa(pmConf: PmUrlConfig, env: Env, messageType: MessageType): HttpUrl {
 
-        val pathSegment = if (messSubCat == OTT) "ccpa_ott/index.html" else "ccpa_pm/index.html"
+        val pathSegment = when (messageType) {
+            LEGACY_OTT -> "ccpa_ott/index.html"
+            OTT -> "native-ott/index.html"
+            MOBILE -> "ccpa_pm/index.html"
+        }
 
         return HttpUrl.Builder()
             .scheme("https")
             .host(env.pmHostCcpa)
             .addPathSegments(pathSegment)
             .addQueryParameter("site_id", pmConf.siteId)
             .addQueryParameter("preload_consent", true.toString())
+            .addQueryParameter("is_ccpa", true.toString())
```

Comment:
Is this query param necessary?

Reply:
@mguerra10128 wrote in a comment in the ticket to add this query param for the ccpa

Reply:
Thanks @carmelo-iriti , just checked the rendering app's code base and it seems to be used by the OTT messages https://github.com/search?q=repo%3ASourcePointUSA%2Fmessage-v2-preact%20isCcpa&type=code